onnxruntime/core/mlas/lib/mlasi.h

```diff
@@ -667,16 +667,9 @@ MlasGetMaximumThreadCount(
 {
 #ifdef MLAS_NO_ONNXRUNTIME_THREADPOOL
     MLAS_UNREFERENCED_PARAMETER(ThreadPool);
+	return onnxruntime::concurrency::ThreadPool::NumThreads(nullptr);
```

Comment:
Here MLAS_NO_ONNXRUNTIME_THREADPOOL means he want to build without ORT.

Reply:
I updated the header and verified that my private scenario works as expected.

Reply:
Thanks for fixing @tracysh